src/scripts/import-open-api.ts

```diff
@@ -468,24 +470,73 @@ export interface ${pascal(name)}Response ${type}`;
   );
 };
 
+/**
+ * Validate the spec with ibm-openapi-validator (with a custom pretty logger).
+ *
+ * @param schema openAPI spec
+ */
+const validate = async (schema: OpenAPIObject) => {
+  // tslint:disable:no-console
+  const log = console.log;
+
+  // Catch the internal console.log to add some information if needed
+  let haveOpenAPIValidatorOutput = false;
+  console.log = (...props: any) => {
+    haveOpenAPIValidatorOutput = true;
+    log(...props);
+  };
+  const { errors, warnings } = await openApiValidator(schema);
+  console.log = log;
+  if (haveOpenAPIValidatorOutput) {
+    log("More information: https://github.com/IBM/openapi-validator/#configuration");
+  }
+  if (warnings.length) {
+    log(chalk.yellow("(!) Warnings"));
+    warnings.forEach(i =>
+      log(
+        chalk.yellow(`
+Message : ${i.message}
+Path    : ${i.path}`),
+      ),
+    );
+  }
+  if (errors.length) {
+    log(chalk.red("(!) Errors"));
+    errors.forEach(i =>
+      log(
+        chalk.red(`
+Message : ${i.message}
+Path    : ${i.path}`),
+      ),
+    );
+  }
+  // tslint:enable:no-console
```

Comment:
?

Reply:
I've disabled the `no-console` rule for the block, so I re-enable it ^^

Reply:
![image](https://user-images.githubusercontent.com/1761469/57936335-fcfe7080-78c3-11e9-9357-0ea2051bfcf9.png)